### ISSUE TITLE
nView Live - Mesh edit mode

### ISSUE THREAD

**Issue by semagnum** [semagnum/nview-live#14]
As a customer, I would like to use the same functionality of nView Live in mesh edit mode for a single mesh. Budget types would be vertex and triangle only.

**Comment by semagnum** [comment]
Technical details: while I could use `bmesh` in edit mode, it can be unstable if the user is editing the mesh - if any vertices or geometry data changes without manually re-checking, especially the vertex order - Blender can fully crash from that alone. I could simply lock the user from making mesh edits during the modal, but then that would kind of defeat the purpose of being in edit mode, wouldn't it?

In short: not sure if this is feasible with Blender's current Python API and mesh structure, at least for it to satisfy the ideal usage of it. Other developers are free to offer suggestions or implementations. In the meantime, I'll leave this issue here for a while longer. If nothing comes of it, I may close the issue as undoable until further notice.
